extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: "Quarkus Extension for Spring Security API"
+metadata:
+  keywords:
+  - "spring-security"
+  - "spring"
```

Comment:
Maybe add `security` as a keyword too.

Reply:
yep, good idea.

Reply:
I just force-pushed an amended commit with the `security` keyword added. Let's wait for CI.